src/porepy/__init__.py

```diff
@@ -196,23 +198,24 @@
     ReferenceVariableValues,
 )
 from porepy.compositional.base import Component, Phase, Fluid
-from porepy.compositional.compositional_mixins import CompositionalVariables, FluidMixin
+from porepy.compositional.compositional_mixins import *
 
 # "Primary" models
-from porepy.models import fluid_mass_balance, momentum_balance
+from porepy.models import energy_balance, fluid_mass_balance, momentum_balance
 
 # "Secondary" models inheriting from primary models
 from porepy.models import (
     poromechanics,
-    energy_balance,
     mass_and_energy_balance,
     thermoporomechanics,
 )
+from porepy.models import compositional_flow
 from porepy.models.fluid_mass_balance import SinglePhaseFlow
 from porepy.models.momentum_balance import MomentumBalance
 from porepy.models.poromechanics import Poromechanics
 from porepy.models.thermoporomechanics import Thermoporomechanics
 from porepy.models.mass_and_energy_balance import MassAndEnergyBalance
+from porepy.models.compositional_flow import ModelSetupCF, ModelSetupCFF
```

Comment:
No particular reason. Can change it to `CompositionalFlow` and `CompositionalFractionalFlow`.
But it must remain somehow clear that these are not *complete* models, they cannot run straight away like `Thermoporomechanics` since a fluid needs to be defined and a constitutive closure. (This was most likely my thinking at that time)

Reply:
I think we agreed to improve the documentation of the ModelSetup clases? Beyond this, I think we can keep the names for now, or change if you have better ideas, then revisit when the full models are ready.

Feel free to resolve when you feel done.